permission.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<style>
```

Comment:
Put style in separate file of css and add those in html. Don't write whole bunch of css code in html.